Makefile

```diff
@@ -0,0 +1,35 @@
+DATE		= $(shell date)
+PYTHON		= $(shell which python)
+NOSETESTS = $(shell which nosetests)
```

Comment:
got it. I usually choose py.test because it more actively developed and nose had its last version released on 2015-06-02